src/Glacier/GlacierClient.php

```diff
@@ -141,17 +147,29 @@ public static function applyDocFilters(array $api, array $docs)
         // Add the SourceFile parameter.
         $docs['shapes']['SourceFile']['base'] = 'The path to a file on disk to use instead of the body parameter.';
         $api['shapes']['SourceFile'] = ['type' => 'string'];
-        $api['shapes']['UploadArchiveInput']['members']['SourceFile'] = ['shape' => 'SourceFile'];
-        $api['shapes']['UploadMultipartPartInput']['members']['SourceFile'] = ['shape' => 'SourceFile'];
+        $api['shapes']['UploadArchiveInput']['members']['sourceFile'] = ['shape' => 'SourceFile'];
+        $api['shapes']['UploadMultipartPartInput']['members']['sourceFile'] = ['shape' => 'SourceFile'];
 
-        // Add information about "checksum" being optional.
-        $docs['shapes']['checksum']['append'] =
-            '<div class="alert alert-info">The SDK will compute this value '
+        // Add the ContentSHA256 parameter.
+        $docs['shapes']['ContentSHA256']['base'] = 'A SHA256 hash of the content of the request body';
```

Comment:
Can you mention that this is calculated automatically? Most users will and should ignore this parameter so I want to make sure the docs are clear.